node/setup.go

```diff
@@ -131,8 +131,14 @@ func logNodeStartupInfo(state sm.State, pubKey crypto.PubKey, logger log.Logger,
 	}
 }
 
-func onlyValidatorIsUs(state sm.State, pubKey crypto.PubKey) bool {
+func hasMajorityVotingPower(state sm.State, pubKey crypto.PubKey) bool {
```

Comment:
This could simply be:
```
// returns true if a pubkey has the majority voting power
func hasMajorityVotingPower(state sm.State, pubKey crypto.PubKey) bool {
    for _, val := range state.Validators.Validators {
        // only one validator can possible have majority, so we only need to check the first instance
        if val.VotingPower > state.Validators.TotalVotingPower()*2/3 {
            return pubKey != nil && bytes.Equal(pubKey.Address(), val.PubKey.Address())
        }
    }
    return false
}
``